docs/tables/github_issue_comment.md

```diff
@@ -0,0 +1,43 @@
+# Table: github_issue_comment
+
+The `github_issue_comment` table can be used to query comments from a specific issue.
+
+**You must specify `repository_full_name` (repository including org/user prefix) and `number` (of the issue) in the WHERE or JOIN clause.**
+
+## Examples
```

Comment:
For this table and the other one, can you please reduce the number of columns in the first example and add a few others?

Reply:
https://github.com/turbot/steampipe-plugin-github/pull/281/commits/68a788b332754d741d2007b449961f958f948a86